src/scenarios/mvcapptemplate/pre.py

```diff
@@ -0,0 +1,17 @@
+'''
+pre-command
+'''
+import sys
+from performance.logger import setup_loggers
+from shared.precommands import PreCommands
+from shared import const
+from test import EXE_NAME
```

Comment:
I think I called this `EXENAME` in the ones I added. Can you change either yours or mine to be consistent?

Reply:
Thought that's the naming convention in python... I'll change mine to EXENAME

Reply:
My take on PEP8 is that it's optional if it improves readability. (I do use that sometimes.)